src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -1,65 +1,40 @@
 package juja.sqlcmd;
 
-import org.junit.After;
-import org.junit.AfterClass;
 import org.junit.Before;
-import org.junit.BeforeClass;
 import org.junit.Test;
 
 import java.sql.Connection;
-import java.sql.DriverManager;
 import java.sql.SQLException;
-import java.sql.Statement;
 
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
-public class DatabaseManagerTest {
+public abstract class DatabaseManagerTest {
     private static final String NAME_DB_TEST = "sqlcmd_test";
     private static final String LOGIN_FOR_TEST = "sqlcmd";
     private static final String PASSWORD_FOR_TEST = "sqlcmd";
-    private static final String LOGIN_FOR_SU = "postgres";
-    private static final String PASSWORD_FOR_SU = "postgres";
-    private static final String TEST_DB_CONNECTION_URL = "jdbc:postgresql://localhost:5432/";
-    private static final String TABLE_NAME = "table_name";
 
-    private static Connection connection;
+    DatabaseManager databaseManager;
 
-    private DatabaseManager databaseManager;
-
-    @BeforeClass
-    public static void createConnection() throws SQLException, ClassNotFoundException {
-        Class.forName("org.postgresql.Driver");
-        connection = DriverManager.getConnection(TEST_DB_CONNECTION_URL, LOGIN_FOR_SU, PASSWORD_FOR_SU);
-        try (Statement statement = connection.createStatement()) {
-            statement.execute(String.format("CREATE DATABASE \"%s\" OWNER \"%s\"", NAME_DB_TEST, LOGIN_FOR_TEST));
-        }
-        connection.close();
-        connection = DriverManager.getConnection(TEST_DB_CONNECTION_URL + NAME_DB_TEST, LOGIN_FOR_TEST, PASSWORD_FOR_TEST);
-    }
+    static Connection connection;
+    static final String TABLE_NAME = "table_name";
```

Comment:
Please rearrange fields declarations because they should be in some order like: constants, final, regular fields